src/gallium/drivers/tegra/tegra_context.c

```diff
@@ -45,16 +45,12 @@ static int tegra_channel_create(struct tegra_context *context,
 static void tegra_channel_delete(struct tegra_channel *channel)
 {
 	tegra_stream_destroy(&channel->stream);
+	drm_tegra_channel_close(channel->stream.channel);
 	FREE(channel);
 }
 
 static void tegra_channel_flush(struct tegra_channel *channel)
 {
-	int err;
-
-	err = tegra_stream_flush(&channel->stream);
-	if (err < 0)
-		fprintf(stderr, "tegra_stream_flush() failed: %d\n", err);
 }
```

Comment:
Ah, yeah. It can be removed for now. I think we can't implement the unified flushing yet as we can't synchronize the 2d-3d streams due to a lack of waitcheks/multiple syncpoints support in the DRM driver.